modules/ROOT/pages/troubleshoot-saml-assertions-task.adoc

```diff
@@ -24,14 +27,6 @@ anypoint.mulesoft.com/accounts/login/<domain_name>
 ----
 +
 
-Click on the specific row that has the above mentioned URL. In the Form data window on the right select the Params tab and copy the SAML response element.
-
-. Log in to the MuleSoft Support Portal at the following URL:
-+
-----
-https://anypoint.mulesoft.com/accounts/login/receive-id
-----
-
 . Click on the specific row that contains the URL mentioned above.
```

Comment:
Line 30 leave out "on" 
Line 32 UI elements should be tagged bold. **Form data** and **Params**